From cdb40a21dd3fbfef2f0104401ce14c4744552e9a Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Tue, 14 Aug 2018 16:39:54 +0800
Subject: [PATCH 020/105] arm64: allwinner: dts: a64: add trigger for TERES-I
 keyboard lock LEDs

The two GPIO-attached LEDs are labelled as capslock/numlock LEDs on the
TERES-I.

Set the default LED trigger of the LEDs to the corresponding lock state
of the keyboard.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-teres-i.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-teres-i.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-teres-i.dts
index f5df5f705b72..d0ac36eab673 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-teres-i.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-teres-i.dts
@@ -55,11 +55,13 @@ leds {
 		capslock {
 			label = "teres-i:green:capslock";
 			gpios = <&pio 2 7 GPIO_ACTIVE_HIGH>; /* PC7 */
+			linux,default-trigger = "kbd-capslock";
 		};
 
 		numlock {
 			label = "teres-i:green:numlock";
 			gpios = <&pio 2 4 GPIO_ACTIVE_HIGH>; /* PC4 */
+			linux,default-trigger = "kbd-numlock";
 		};
 	};
 
-- 
2.27.0

